mono/ExtensionsPath-ToolsVersion/Microsoft.Common.targets/ImportBefore/Microsoft.Common.Mono.Before.targets

```diff
@@ -0,0 +1,24 @@
+﻿<!--
+***********************************************************************************************
```

Comment:
This needs to be moved somewhere else not off the root. Maybe in `src/Tasks/Mono/Microsoft.Common.Mono.Before.targets`?

Reply:
I wanted to keep the mono-specific files[1] under `mono/`, like https://github.com/mono/msbuild/tree/xplat-master/mono . I will move the other mono-specific targets to this kinda folder and use `ImportBefore` (and similar) hooks to import then, instead of hardcoding the import in the main targets files.

1. mono-specific files like any additional targets, tasks, build scripts etc, to keep them cleanly away from rest of the msbuild code